netlify.toml

```diff
@@ -0,0 +1,26 @@
+[build]
+publish = "public/"
+command = "hugo -b $DEPLOY_PRIME_URL"
+
+[build.environment]
+# added specifically a docker image, so the versions within makefile and netlify documentation match
+# renovate: datasource=docker depName=klakegg/hugo
+HUGO_VERSION = "0.107.0"
+HUGO_ENABLEGITINFO = "true"
+
+# On netlify our branch will always be the one we are currently building for
+# important information regarding naming 
+# https://gohugo.io/getting-started/configuration/#configure-with-environment-variables
+HUGOxPARAMSxGITHUB_BRANCH="$BRANCH"
```

Comment:
@aepfli in KLT this seemed not to work, we had to explicitly pass it as an env var to the hug command, [here]( https://github.com/keptn/lifecycle-toolkit/blob/main/netlify.toml#L30)